### ISSUE TITLE
ParseDirectory fails under Windows

### ISSUE THREAD

**Issue by morriswinkler** [asticode/go-astitools#1]
Just debugged this one

Since the template name is trimmed to /index.html 
what results on windows in \index.html. The template isn't found later since the names differ.

```
DEBU[0000] serve templates: &{name:\index.html Tree:0xc042001400 common:0xc042102900 leftDelim: rightDelim:}
DEBU[0000] serve templates: &{name:\train.html Tree:0xc042001500 common:0xc042102900 leftDelim: rightDelim:}
DEBU[0000] filepath : C:\Users\IEUser\go\src\kmine\resources\templates
DEBU[0000] Listening on 127.0.0.1:51090
DEBU[0000] Sending to Astilectron: {"name":"window.cmd.create","targetID":"1","url":"http://127.0.0.1:51090/templates/index","windowOptions":{"backgroundColor":"#333","center":true,"height":600,"title":"Kmine","width":600}}
DEBU[0000] Astilectron says: {"name":"window.event.focus","targetID":"1"}
DEBU[0002] got request : [{Key:page Value:/index}]
DEBU[0002] not found : /index.html
```

I propose 

```
                // Parse template
		tName := strings.TrimPrefix(path, i)
		tName = strings.Replace(tName, `\`, `/`, 1) // flip windows leading path
		var c = t.New(tName) 

```

as a fix, i guess i will send a PR soon for that

**Comment by asticode** [comment]
I think using `filepath.ToSlash` would be a more suitable solution. I've pushed the change. Does it fix your problem?

**Comment by morriswinkler** [comment]
`12345=-0987654321

**Comment by morriswinkler** [comment]
fixed with 9b6911ee1cf3f19495a53d236aebbdcef930ea3b
